stream.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Oscillator stream
+ * http://webaudio.github.io/web-audio-api/#the-oscillatornode-interface
+ *
+ * @module  audio-oscillator
+ */
+'use strict';
+
+var direct = require('./direct')
+var Readable = require('readable-stream')
```

Comment:
with _readable-stream_ vs _audio-through_ there is a bunch of problems which are not necessarily addressed/covered. Within others, stack size (for RT should be `0`), ending stream (if source/target stream ends and other cases), converting input chunk to _AudioBuffer_ (it may be raw buffer), etc.